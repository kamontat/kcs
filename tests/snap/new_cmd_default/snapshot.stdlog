[ERR] exit.base            missing 'commands:_empty'
