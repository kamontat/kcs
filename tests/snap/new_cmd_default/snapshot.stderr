[ERR] exit.base            missing 'commands:_empty' ($KCT_PATH_TESTDIR/commands/_empty.sh $KCS_PATH_ROOT/commands/_empty.sh $KCS_PATH_SRC/commands/_empty.sh)
